easybuild/easyblocks/generic/binary.py

```diff
@@ -98,6 +98,11 @@ def install_step(self):
         """Copy all files in build directory to the install directory"""
         if self.cfg['install_cmd'] is None:
             try:
+                cmd = self.cfg['preinstallopts']
+                if cmd:
+                    self.log.info("Running preinstallopts of %s using command '%s'..." % (self.name, cmd))
+                    run_cmd(cmd, log_all=True, simple=True)
+                self.log.info("Copying %s into '%s'..." % (self.cfg['start_dir'], self.installdir))
```

Comment:
hmm, this is not how `preinstallopts` is usually being used

not that this also mismatches the name, these are not 'options', but an actual command (granted, something like `foo &&` isn't an option either, `pre*opts` gets sort of abused easily for that)